pages/index.js

```diff
@@ -1,9 +1,112 @@
-import EmotionList from "@/components/EmotionList";
+import styled from "styled-components";
+import useLocalStorageState from "use-local-storage-state";
+import { uid } from "uid";
+import Link from "next/link";
+import { date } from "@/utils/date-constructor";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 80vw;
+  margin: 2rem auto;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledLabel = styled.label`
+  padding: 2rem;
+  text-align: center;
+`;
+
+const StyledInput = styled.input`
+  width: inherit;
+`;
+
+const StyledWrapper = styled.div`
+  display: flex;
+  width: inherit;
+  justify-content: space-between;
+`;
+
+const StyledSpan = styled.span`
+  font-size: 0.8rem;
+`;
+
+const StyledButton = styled.button`
+  width: 3rem;
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 2rem auto;
+  text-align: left;
+`;
+
+const StyledListItem = styled.li`
+  margin: 1.2rem auto;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  padding: 1rem;
+  background-color: lightgray;
+  border: 1px solid black;
+  border-radius: 5px;
+  color: black;
+`;
 
 export default function HomePage() {
+  const [tensionEntry, setTensionEntry] = useLocalStorageState("tensionEntry", {
+    defaultValue: [],
+  });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    // variable date imported from util/date-constructor.js
+    setTensionEntry(
+      tensionEntry.length === 0
+        ? [{ ...data, id: uid(), date: date }]
+        : [...tensionEntry, { ...data, id: uid(), date: date }]
+    );
+  }
+
   return (
     <>
-      <EmotionList />
+      <StyledForm onSubmit={handleSubmit}>
+        <StyledLabel htmlFor="tension-level">
+          On a scale from 0 to 100, how tense do you feel in this moment?
+        </StyledLabel>
+        <StyledInput
+          id="tension-level"
+          name="tensionLevel"
+          type="range"
+          defaultValue={0}
+          max={100}
+        ></StyledInput>
```

Comment:
It is perfectly valid in JSX to use a explicit closing tag, but for consistency and brevity, especially with single-tag elements, it's common to use self-closing tags.
```suggestion
        <StyledInput
          id="tension-level"
          name="tensionLevel"
          type="range"
          defaultValue={0}
          max={100}
        />
```